### ISSUE TITLE
Traefik v2.0.6 + Rancher 1.6.29 UI not pageable

### ISSUE THREAD

**Issue by maxifom** [containous/traefik#5959]
### Do you want to request a *feature* or report a *bug*?
Bug

### What did you do?
Started Traefik v2.0.6 with Rancher 1.6.29.
Tried to select next page 
### What did you expect to see?
Next page of view

### What did you see instead?
![Screenshot from 2019-12-08 00-49-29](https://user-images.githubusercontent.com/20924207/70380053-26154a00-1957-11ea-942f-3128e734378a.png)

### Output of `traefik version`: (_What version of Traefik are you using?_)
```
Version:      2.0.6
Codename:     montdor
Go version:   go1.13.4
Built:        2019-12-02T17:24:31Z
OS/Arch:      linux/amd64
```
### What is your environment & configuration (arguments, toml, provider, platform, ...)?
Traefik dashboard is accessed by https and is proxies through traefik.
I have 16 http routers, which is more than 10 per page. So I want to see second page, but UI doesn't allow me.

**Comment by ldez** [comment]
hello,

In the v2.0.x, the list (of routers, services, ...) are not refreshed, could you try to refresh manually?

**Comment by ClovisGauzy** [comment]
Not Rancher related : same problem here with Docker [Swarm] provider.

**Comment by ldez** [comment]
@ClovisGauzy could you provide more information?

**Comment by ClovisGauzy** [comment]
What would you like ? ^^

**Comment by ClovisGauzy** [comment]
As I can see from DOM level, the navigations buttons don't have event attached to, just a `cursor: not-allowed !important` CSS rule.

**Comment by ldez** [comment]
could you provide more information about your context (how many routers, services, ...)

**Comment by maxifom** [comment]
> hello,
> 
> In the v2.0.x, the list (of routers, services, ...) are not refreshed, could you try to refresh manually?

The issue is not about that the list is not being refreshed, but that I cannot get to the second page.

**Comment by ClovisGauzy** [comment]
@ldez 
29 HTTP Routers
5 HTTP Services
10 HTTP Middlewares
No TCP Routing (for the moment)

The problem looks more like a Vue Component incomplete issue : pagination is stuck to `limit: 10` and it's impossible to go page 2 and more.

**Comment by traefiker** [comment]
Closed by #5986.
